script.js

```diff
@@ -1,10 +1,12 @@
 const quizQuestions = [];
 
+const sectionEl = document.querySelector('.question-section');
 const formEl = document.getElementById('quiz-form');
 const submitBtn = document.getElementById('submit-btn');
 const questionInput = document.getElementById('question');
 const answerInputs = document.querySelectorAll('.answer'); //get NodeList
 const correctnessInputs = document.getElementsByName('correctAnswer'); //get NodeList
+const randomizeBtn = document.getElementById('randomize-btn');
 
 formEl.addEventListener('submit', submitForm);
```

Comment:
Great to see you have created a submitForm function and calling it in this eventListener rather than writing all the code directly in it!

Reply:
Thank you for your feedback!